tasks/move-granules/schemas/config.json

```diff
@@ -0,0 +1,63 @@
+{
+  "title": "MoveGranulesConfig",
```

Comment:
How much of this is actually needed / used by the task?  Couldn't / shouldn't it just be properties for source, destination, and granule ID?